.k8s.mk

```diff
@@ -15,7 +15,7 @@
 K8S_CONF_DIR = k8s/conf/
 
 # Need nsmdp and icmp-responder-nse here as well, but missing yaml files
-DEPLOY_NSM = nsmd vppagent-dataplane
+DEPLOY_NSM = jaeger nsmd vppagent-dataplane
```

Comment:
Could we move jaeger to DEPLOY_INFRA ? similar to what is done with skydive via DEPLOY_MONITOR... perhaps a DEPLOY_TRACING?